ThemingCodelab/gradle/libs.versions.toml

```diff
@@ -0,0 +1,127 @@
+#####
```

Comment:
we don't use version catalogs in codelabs, we hardcode the versions, as we don't use the same automatic update process like in the samples